packages/modules/manage-versions/tsconfig.json

```diff
@@ -1,14 +1,12 @@
 {
   "compilerOptions": {
-    "target": "es2017",
+    "target": "ES2020",
     "lib": [
-      "esnext.asynciterable",
-      "es2017",
-      "dom",
-      "dom.iterable",
-      "scripthost"
+        "dom",
+        "dom.iterable",
+        "esnext"
     ],
-    "module": "esnext",
+    "module": "ES2020",
```

Comment:
why downgraded?

Reply:
Just specifically targeting es2020 to be consistent. Maybe we can have a separate issue to change it to latest

Reply:
to be consistent with what?
previously we had `esnext` and that should be fine.. Less changes - better.

Reply:
Consistent with the targetted es2020 but since ES and modular system works mostly independent of each other so reverted them back to esnext